collectd-cpu/meta.yaml

```diff
@@ -1,7 +1,7 @@
-{ "display_name":"Aggregation",
-  "description": "SignalFx uses the aggregation plugin for collectd to aggregate per-CPU metrics into overall sum and average per host.",
-  "project_url": "https://github.com/signalfx/integrations/tree/master/collectd-aggregation",
-  "code": "https://github.com/signalfx/collectd/blob/master/src/aggregation.c",
+{ "display_name":"CPU",
+  "description": "SignalFx uses the collectd cpu plugin for collectd to get CPU metrics per host.",
```

Comment:
redundant "the collectd cpu plugin for collectd"

Reply:
Fixed.